tindergames/src/main/java/org/teamcifo/logic/UserManager.java

```diff
@@ -0,0 +1,52 @@
+package org.teamcifo.logic;
+
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.AllArgsConstructor;
+import lombok.Setter;
+import java.util.HashMap;
+import java.util.Map;
+import org.teamcifo.domain.*;;
+
+
+@Data
+@AllArgsConstructor
+@Getter
+@Setter
+public class UserManager{
+
+    private Map<String, User> users;
+
+    public UserManager(){
+        this.users= new HashMap<>();
+    }
```

Comment:
As explained in the first comment, `UserManager` can't be instantiated, this would lead to multiple `UserManager` objects, each one with its own users database

Reply:
This class must be refactored. Do it in another task